cajita/src/Cajita_ManualPartitioner.hpp

```diff
@@ -28,17 +28,14 @@ namespace Cajita
 /*!
   \brief Backwards compatibility wrapper for 3D ManualBlockPartitioner
 */
-class ManualPartitioner : public ManualBlockPartitioner<3>
-{
-  public:
-    /*!
-      \brief Constructor
-      \param ranks_per_dim MPI ranks per dimension.
-    */
-    ManualPartitioner( const std::array<int, 3>& ranks_per_dim )
-        : ManualBlockPartitioner<3>( ranks_per_dim )
-    {
-    }
+class [[deprecated]] ManualPartitioner : public ManualBlockPartitioner<3>{
```

Comment:
Format looks wrong

Reply:
Agreed, but I think it's the deprecation tag messing it up - not sure if we can fix it easily in the format options